common/constants/default.go

```diff
@@ -14,6 +14,8 @@ const (
 	// NodeName is for the clearer log of cloudcore.
 	NodeName = "NodeName"
 
+	CommonName = "KubeEdge"
```

Comment:
Is this definition OK? @fisherxu @zc2638

Reply:
like `ProjectName` ? Or other better @gy95

Reply:
yes, this is better. `CommonName` is a little not specific.

Reply:
I have modified it

Reply:
/LGTM, please squash the commits to one @dsy3502

Reply:
I have fixed it

Reply:
/lgtm